src/platform/graphics/uniform-buffer.js

```diff
@@ -284,6 +233,46 @@ class UniformBuffer {
         } else {
             this.storageFloat32 = null;
             this.storageInt32 = null;
+            this.storageUint32 = null;
+        }
+    }
+
+    getStorageForType(type) {
```

Comment:
I suspect this is compiled to an array lookup by most browsers, but I'd prefer we don't rely on this and avoid executing this for each uniform we set - that could be costly.